### PR TITLE
add vertical-pod-autoscaler images

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/14557

For new image repos read the docs to create repos in our registries:
https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/container-registry/

If you're an admin you can just ping SIG releng and force merge your PR once CI is green.

---

### CODE REVIEW

**Comment by TheoBrigitte** [comment]
@giantswarm/sig-releng do I need to create new registries manually?

**Review by omegas27 — APPROVED** [review]
LGTM

File: images.yaml [hunk 1]
```diff
@@ -582,6 +591,10 @@
   tags:
   - sha: 1fad3f247a8edeceead03230d833fcb4c3935a3a57b83787690f15f7daa6b59b
     tag: v5.1.0
+- name: quay.io/reactiveops/ci-images
```
Comment (MarcelMue) [inline images.yaml:594]: What is this?

**Review by MarcelMue — COMMENTED** [review]
Shouldn't need to create registries

File: images.yaml (hunk 1 above)
Comment (TheoBrigitte) [reply images.yaml:594]: This image is used to generate the certificates for the admission controller.

Comment (omegas27) [reply images.yaml:594]: This image is used by VPA to generate certificates for the admission controller and is used internally by the helm chart for testing

**Comment by TheoBrigitte** [comment]
> Shouldn't need to create registries

Ok then let's delete this PR comment, because it's very confusing.

**Comment by MarcelMue** [comment]
Let's try it first - maybe I misremember. Quay is for sure fine but aliyun is only fine _in my head_ - I can't find docs

**Comment by TheoBrigitte** [comment]
> Let's try it first - maybe I misremember. Quay is for sure fine but aliyun is only fine _in my head_ - I can't find docs

Is this an approval ?

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**